source/DasBlog.Web.Repositories/Interfaces/ISiteSecurityRepository.cs

```diff
@@ -5,38 +5,38 @@
 
 namespace DasBlog.Web.Repositories.Interfaces
 {
-    public interface ISiteSecurityRepository
-    {
-        bool IsValidContributor { get; }
+	public interface ISiteSecurityRepository
+	{
+		bool IsValidContributor { get; }
 
-        bool IsInRole(string role);
+		bool IsInRole(Role? role);
```

Comment:
Should this be nullable for this method? We actually we to test a value.

Reply:
I'm gonna change this around to use attributes on the Controller actions.